package.json

```diff
@@ -51,8 +51,9 @@
     "build": "babel --ignore=__tests__ src --out-dir lib",
     "lint": "standard",
     "prepublish": "npm test",
-    "posttest": "NODE_ENV=production npm run build",
+    "pretest": "npm run build",
```

Comment:
I wanna make sure I understand what you're saying... if you remove `pretest`, the tests don't catch type errors?